src/components/Layout.js

```diff
@@ -19,12 +16,11 @@ const Layout = ({ pageTitle, children }) => {
   `)
 
   return (
-    <div className="container">
+    <div>
       <title className="heading">{pageTitle} | {data.site.siteMetadata.title}</title>
```

Comment:
I removed some of this in my PageTitle component commit - will it be an issue?

Reply:
No issue and thanks for the heads up!